distribution/ddf-common/src/main/resources/etc/shell.init.script

```diff
@@ -56,6 +56,7 @@ if { %(jlineReader != null) } {
 
   setopt group
   setopt auto-fresh-line
+  unsetopt insert-tab
```

Comment:
On line 79 in the `DDF additions` section, we added `setopt no-insert-tab`. Does that conflict with this line?

Reply:
Doesn't seem to conflict but it looks like our addition isn't needed either so I removed it.